### PR TITLE
Change to cloudwatch logging structure to resemble similar to ROS

### PR DESCRIPTION
**Pull request by kusumachalasani**
The current code supports cloudwatch logging structure as below:

```
 "logging": {
        "cloudwatch": {
          "accessKeyId": "",
          "logGroup": "kruize-logs",
          "logStream": "kruize-stream",
          "region": "",
          "secretAccessKey": "",
          "logLevel": "INFO"
        }
      }
```

### CODE REVIEW

File: src/main/java/com/autotune/utils/CloudWatchAppender.java [hunk 1]
```diff
@@ -106,6 +109,7 @@ public static void configureLoggerForCloudWatchLog() {
                 LoggerConfig loggerConfig = config.getLoggerConfig("com.autotune");
                 loggerConfig.addAppender(appender, level, filter);
                 context.updateLoggers(config);
+                LOGGER.info("Enabled CloudWatch logs");
```
Comment (khansaad) [inline src/main/java/com/autotune/utils/CloudWatchAppender.java:112]: Should we make it `debug` ?

**Review by khansaad — APPROVED** [review]
LGTM!

**Review by dinogun — APPROVED** [review]
LGTM

File: design/KruizeConfigurationProcedure.md [hunk 2]
```diff
@@ -49,13 +49,15 @@ data:
...
         "showsql": "true"
       },
-      "cloudwatch": {
-        "accessKeyId": "",
-        "secretAccessKey": "",
-        "region": "",
-        "logGroup": "kruize-logs",
-        "logStream": "kruize-stream",
-        "logLevel": "INFO"
+      "logging": {
```
Comment (chandrams) [inline design/KruizeConfigurationProcedure.md:52]: @kusumachalasani - Manifest yamls don't need an update to include logging?

Comment (kusumachalasani) [reply design/KruizeConfigurationProcedure.md:52]: yes, they need them. let me create an another PR!

**Pull request merged**